extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/Chart/ChartWrapper.tsx

```diff
@@ -130,7 +130,10 @@ export const ChartWrapper = ({
             }
             const metricObj: ChartDataProps = {
               ...obj,
-              name: obj?.metric && typeof obj?.metric?.pod === 'string' ? obj?.metric?.pod: Object.values(obj?.metric).join(":"),
+              name:
+                obj?.metric && typeof obj?.metric?.[groupBy] === "string"
```

Comment:
name is based on the metricName  config 
https://github.com/argoproj-labs/argocd-extension-metrics/blob/main/app/config.json#L44